src/components/mycourses/courseDetails/index.vue

```diff
@@ -10,16 +10,25 @@
       <!--  <a-button type="danger" size="large" @click="setAllUnavailable">
         Close Course
       </a-button> -->
-      <a-tooltip
-        placement="topLeft"
-        title="Caution: All individual classes will be closed!"
-        arrow-point-at-center
-      >
-        <closeCourseAndSeminarsToggle
-          v-bind:course_group_id="course_group.id"
-        />
-      </a-tooltip>
     </div>
+    <a-tooltip
+      placement="topLeft"
```

Comment:
- I suggest you put some kind of warning icon like exclamation mark beside the toggle button. And the tooltip appears over the icon.
- currently the tooltip appears to the right of the screen, which is abit weird.
- I think the wording "will apply the same effect" is not as clear as it can be. I suggest: "Closing/opening this course group will automatically close/open all of its classes."
- currently when we close a course group or an individual class, the phrase "This class is open to requests" does not change to "This class is closed to requests." Same for "This course is closed to requests".
- I suggest to add the word visit: "This class is open to visit requests", "This class is closed to visit requests".
- I suggest to use the term "course_group" instead of "course". "This course group is open to visit requests".